src/app/ui/landing-page/landing-page.js

```diff
@@ -172,7 +172,8 @@ export class LandingPage extends ViewPlugin {
AIWgzAegOIyw6STVbIx6z1f+MI+RsTHihEBqTQjbRPMwtVY8UwwFktFYtXScgD9fY75665VhNUzZo/6FTReHBwoQhd1aDFYUZmGslSssinwhA29XUPjAApbPcLJI268EjY0LDDSrNFj/M/J2ORlTeHHFIijWclDxsHjwN6Ow38BzD9h4v9euHj0EyfauertAfJ7Geqpltzy7/0pfh/PnLbqNgCXEfAKMd89UkObp0z/z6vA+lsAV5aVS+abWpoa7pcVUgRTKMPG9f1/ZVnqC2CaCyMSCtQqrJiMpUGVN4cxaMZPKFzQY1jIFz0rOS4fZQI4kM1a/zFzVl2agZuIUEUEq7Sy0xRpI6rU0ZNqHY9KA31XCshmvWkkZkQZb02JzOCZgZ4eB9uaDsSuL9jSy7KcXTtzXs2Ezv39j+siFgCAnVJIpy1YKeX3NXpPpINo3Be9ICUb/D30g431VrLx82iIMzi6HUDMLJ3CNiEj6iejq5ITUaVZQsNogUIrsbp3weLan8q34e3n1LPur9yxZUX/SL6GdwGCvwdwBryvRRFAGkC361hnt7besEs+iZFD9jDfyR/eQPFpzfxNKN4ARY7yh0yD2NvfVJGRe9DkT4SwrzIYSh3KlR9xWTahoiqFmlEZZCvs4D7Ji6vawUE96dXXu75kpzEDGl9hjV2mlWnMA9Vse0hMN0mqMZnCyYm6RIWYx15YdOSnhLs6Qgs8lxQ9ocBLl1wy/vkFl9b29/ai3cnrOcHcylTKgrIIxYJGsajhFBmua4ieawx+9ns8g5YR7TJ00Y8uXZ2INA3rOr+aFkYWIDBD94e2hUJIlLiI8Y0TwEaKlmiPUvQVWHTVwkV17xOxPHEYabEEvCIkIloJoNtfv9P+d2a1bRdt+RQkwhQOw5Y3+i+CZdVD4yoyVChI7emkYgXN+ByN2ILZv2ekUoNIx8l7e5xuaAJARQKvWfT+sZeb76VrPy8D47NQmFcyhJrNsV5RSjZIO+pEmwgIyKYBy4pENZoIYqYwvd2jwUEXW7d0aXgVmjdalvXCRZec1BqGmht7/47BXyUC2bZCZXUaYI2+niK09qqO01kLpCgSa3jFPq7hIxtUuLphRKkNV58oXYtwb5li+5VeNS7FDkTY+sPl7O1oJ0M3M+i3iu1fzl9c/ayc9X/eTJ3+4M3MvALAVAD7FHBlLlf/nBwZEUxhqML5+uBMpNRyxVjG0JVhCtOI35h1iVc2g4y9Nf8WMhf3qH3CdVw4BQ2nUHySSX1nyfvHPFPuvXTt508w42oAlwGBjWm86MeMHIO0J8GbPhJodzoD2HZ0n0AgtVG9qvyf7dzZz/v2DvwCRGsW/8WYkhFRjZv6LtAur7HTND6dtcAacApFFB3vYCiLkK1IAVDhcWLTSMBsCQmGd5upW7/XMnIDCozbKTazMkzHlhilx/7ZQQyHiB7XFv0mRbxx7ntGiT+oEDJl+oMfZq3PUpZqa9q6/Ak5IiKYwtEI55a+U5S2b2TGVQCmatZhD59O5EOT1axmdWawdEdp06DiEzuLrnv3vHeP+sbh3sv+/fwB5eLTUFjKGnYgKCXt+IYhA3ybOwaQTgOpVFxYw/tF+5dcdLh/8+bOu92i+8SS948vW1qfe71nnLboe+lK+wPplEJvdx6ufzwsy4Kd9tyUwBSan2tjvJfW8WHRWiccgTRB+yG0NquQjWjfLNaJhjsTg5m823iHBh4hwmtIp19OF9y+CxbXdMpZLQgimMII0tjIp7n5wlVE+BiYFwRepcFiTUH0GFTMcFInOSpaZaOJnuk+p8j/NXdh9ZBHGe3bx2crxnUAPsmMSUhW0AZ9iqZ+MmBbXuGPmY6FcX8QClrzmv3tA7+YPqvqgcO9jx05nqQt5y7Hca8Y6HcqgsiysspTZa2NySSBGPqtNl7xDoXVsaYwmuIZRJFIRNKsXYSevyAQcT9r2kFK7WLojYqt1y5cVPOQnLmCIIIpvA28ufZAjVtpVSuVPptcvoaZPwWgh5ltMNea1nmRsTvFF3zvqVzWupsUvXjeBVWXH8176eriVF0dOfv2cAN56dr5MHovNeL9hSDAVl5aNuzz9Ds8wDjguthoAbcqIFc9nnYP9X20NjkX9/fl67WrP8aAymQtK5W2lI61kmjP7UezV1XsTwoJrewM83XioM3Fnyyi2U8rR1Wtfjp2Dxi9APaB+CWV5a/pfjVdVao/HC83HkEQRDCFIbJ53cBklaIPMVRagzPE+oMEWqShOwlUzYyMFyAx+XuNfuk6rwP4CVfpNeefX/vmsXgv+9v5A2Bcx8CVzGBfOCNjGD8zXFWFaKyWRj8BjVrjBwB+VTeOXjnqY7G+9zzW+trKqvRHQNjpujyWNc4CAnE0hjgH7SXh2K24OTtABc2cZs0dYIxmot3E6GbCbjB3K6h7HdcZb9vpjdrJd81fMma7nI2CIIIpvNNEdEP/e5h4DrtoU0qdy4xLoPUWWBaxqweg6I1zZ1c8MlKvv3cHzwRhGTOWAnAAKAZOJYImAjIZHLAUOpjRojX+t3YMrTzW76GlsbDQHSzuQFrN0q6+nhk7WGNQa14K4FVX6xowjWPmHGvMYqCVNQ1odicopmehMMVl+jXY1YrtRjA7F1xU8/zLL3dPX7hwlNiVCYIgCMeW9p38sfY2vnr3Tl7W1lz87p4dfN2B/bysp4Nn9fQcP2PplnWdYY/yhrVdUwFg/frBKRs3dmcAYNOrA2cAwNq1eyfKpyYIgiAIgiAIgiAIgiAIgiAIgiAIgiAIgiAIgiAIgiAIgiAIgiAIgiAIgiAIgiAIgiAIgiAIgiAIgiAIgiAIgiAIgiAIgiAIgiAIgiAIgiAIgiAIgiAIgiAIgiAIgiAIgiAIgiAIgiAIgiAIgiAIgiAIgiAIgiAIgiAIgiAIgiAIgiAIgiAIgiAIgiAIgiAIgiAIgiAIgiAIgiAIgiAIgiAIgiAIgiAIgiAIgiAIwnD4fyD8+F98nGKgAAAAAElFTkSuQmCC'
```

Comment:
Could we store the image in an "assets" folder instead of putting the base64 here.
Also consider using webp, this is the best format for the web. You can use this to convert : 
https://squoosh.app/

Reply:
right. we have it there as svg.